common/src/main/scala/azure/APNSRawPush.scala

```diff
@@ -0,0 +1,9 @@
+package azure
+
+import azure.apns.Body
+
+case class APNSRawPush(body: Body, tags: Option[Tags]) {
+  def tagQuery: Option[String] = tags.map { set =>
```

Comment:
this is duplicated for each RawPush probably and is specific to azure not APNS or GCM